src/vmm_core.mli

```diff
@@ -22,48 +22,67 @@ module IM : sig
 end
 
 module Name : sig
-  type t
+  module Label : sig
+    type t
+
+    val compare : t -> t -> int
+    val equal : t -> t -> bool
+    val is_empty : t -> bool
```

Comment:
this binding and the one below, will have to disappear. there's one use in vmm_trie which we can replace by option.